src/components/cart/OrderSummary/OrderSummary.tsx

```diff
@@ -0,0 +1,44 @@
+import type { InputHTMLAttributes, ReactNode } from 'react'
+import React from 'react'
+import { useFormattedPrice } from 'src/sdk/product/useFormattedPrice'
+
+import './order-summary.scss'
+
+export interface OrderSummaryProps extends InputHTMLAttributes<HTMLDivElement> {
+  subTotal: number
+  total: number
+  numberOfItems: number
+  checkoutButton: ReactNode
+}
+
+function OrderSummary({
+  subTotal,
+  total,
+  numberOfItems,
+  checkoutButton,
+}: OrderSummaryProps) {
+  const discount = subTotal - total
+  const formattedDiscount = useFormattedPrice(discount)
+
+  return (
+    <div className="order-summary" data-order-summary>
+      <p data-order-summary-subtotal>
+        <div>Subtotal ({numberOfItems} products)</div>
+        <div>{useFormattedPrice(subTotal)}</div>
+      </p>
```

Comment:
I wonder if we can use an unordered list here! Something like the following (suggestion not to be committed): 

```suggestion
    <ul className="order-summary" data-order-summary>
      <li data-order-summary-subtotal>
        <span>Subtotal ({numberOfItems} products)</span>
        <span>{useFormattedPrice(subTotal)}</span>
      </li>
```

---

This way we can also keep all the common CSS under the `li` styles and avoid the [current repetition](https://github.com/vtex-sites/base.store/blob/411b3b42f83f4aa5cc4d77410cc644fc57fb2a73/src/components/cart/OrderSummary/order-summary.scss#L6-L22).
```scss
li {
    display: flex;
    justify-content: space-between;
    line-height: 1.5;
	&[data-order-summary-discount] { color: green; }
}
```

Reply:
Reminded me of PetPlate's Checkout Summary. 
🙊 

![CleanShot 2022-01-18 at 18 30 39](https://user-images.githubusercontent.com/381395/150021896-62ff568f-f5de-452b-a02f-2bb5d388fa00.png)

cc @hellofanny 💙

Reply:
I suggested the same thing ahahah 💛  , But instead we can use Faststore's List component. What do you think?

Reply:
I've done this using the `List` component. Thanks <3